modules/cli/src/main/scala/scala/cli/ScalaCli.scala

```diff
@@ -194,6 +196,12 @@ object ScalaCli {
       // Enable ANSI output in Windows terminal
       coursier.jniutils.WindowsAnsiTerminal.enableAnsiOutput()
 
+    for {
+      configDb   <- ConfigDb.open(Directories.directories.dbPath.toNIO)
```

Comment:
why not do it straight on `isSipScala` variable initialization?

Reply:
I would be changed in another PR as part of #1824

Reply:
Thanks, so I move this logic to `isSipScala` variable initialization.